### PR TITLE
refresh cat dropdown (bug 1103225)

### PR DESCRIPTION
**Pull request by ngokevin**
screen shot 2015-01-12 at 6 37 51 pm

### CODE REVIEW

File: src/media/css/category-dropdown-desktop.styl [hunk 1]
```diff
@@ -48,121 +61,69 @@
...
-        border-top: 1px solid $light-gray;
-        border-right: 1px solid $light-gray;
+        border-top: 1px solid $tile-border;
+        border-right: 1px solid $tile-border;
         float: left;
         font-size: 15px;
         height: 50px;
         position: relative;
-        text-transform: uppercase;
+        text-transform: none;
```
Comment (spasovski) [inline src/media/css/category-dropdown-desktop.styl]: can probably rm this line

**Comment by spasovski** [comment]
I thought we wanted the hover state of each nested category link to extend to the edges of its parent container the way it once was. Granted I see on -dev it's also similar to this.

r+ in any case. @pwalm you might want to check it out also after it's merged

File: src/media/css/category-dropdown-desktop.styl (hunk 1 above)
Comment (ngokevin) [inline src/media/css/category-dropdown-desktop.styl]: it inherits an uppercase from the navbar :/

**Comment by ngokevin** [comment]
thanks, yeah let's get some further opinions later

**Pull request merged**